(DOCSP 30002): Flutter FTS (#2847)
mongodb/docs-realm: source/sdk/flutter/realm-database/model-data/property-annotations.txt

.. literalinclude:: /examples/generated/flutter/schemas.snippet.property-annotations.dart
   :language: dart
   :emphasize-lines: 12-13

.. _flutter-fts-annotation:

Full-Text Search Indexes
~~~~~~~~~~~~~~~~~~~~~~~~

In addition to standard indexes, Realm also supports Full-Text Search (FTS)
indexes on string properties. While you can query a string field with or without 
a standard index, an FTS index enables searching for multiple words and phrases 
and excluding others.

> For more information on querying full-text indexes, see 
> :ref:`Filter with Full Text Search <flutter-fts>`.

Reviewer: Couple small edits for consistency in how we refer to FTS.
```suggestion
For more information on querying FTS indexes, see 
:ref:`Filter with Full-Text Search <flutter-fts>`.
```

> To create a FTS index on a property, use the :flutter-sdk:`@Indexed 

Reviewer: ```suggestion
To create an FTS index on a property, use the :flutter-sdk:`@Indexed 
```

> <realm_common/Indexed-class.html>` annotation and specify the 

Reviewer: TIL, `:flutter-sdk:` doesn't work for all of the Flutter SDK's docs. Looks like `realm_common`, which `Indexed` is part of, has a different root URL, so this convention results in a 404.

We'll need to use the full URL, like lines 94 and 95 in this file.

> :flutter-sdk:`IndexType <realm_common/RealmIndexType.html>` as ``fullText``. 

Reviewer: Same as above. Right now, this 404s.

This enables full-text queries on the property. In the 

> following example, we mark the genre property with the FTS annotation:

Reviewer: ```suggestion
following example, we mark the pattern and material properties with the FTS annotation:
```
